src/UALJs.ts

```diff
@@ -131,6 +131,8 @@ export class UALJs extends UAL {
     localStorage.setItem(UALJs.SESSION_EXPIRATION_KEY, `${thirtyDaysFromNow.getTime()}`)
     localStorage.setItem(UALJs.SESSION_AUTHENTICATOR_KEY, authenticator.constructor.name)
 
+    await this.isAuthenticatorLoading(authenticator)
```

Comment:
If this function is just to wait for loading, perhaps `await this.waitForLoading(authenticating)` or something like that?

Reply:
👍updated to `waitForAuthenticatorToLoad`